src/store/index.js

```diff
@@ -33,12 +37,30 @@ export default createStore({
       state.apiBaseUrl = url
     },
 
-    setDataSessionsUrl(state, url) {
-      state.dataSessionsUrl = url
-    },
-
     setArchiveUrl(state, url) {
       state.archiveUrl = url
+    },
+    
+    setOcsApiUrlUrl(state, url) {
```

Comment:
The double "url" here, while I get why you did it, seems like it could be confusing

Reply:
that was an accident oopsiiiii